src/messaging/ExchangeContext.h

```diff
@@ -124,24 +126,19 @@ class DLL_EXPORT ExchangeContext : public ReliableMessageContext, public Referen
     /**
      *  Handle a received CHIP message on this exchange.
      *
-     *  @param[in]    packetHeader  A reference to the PacketHeader object.
-     *
+     *  @param[in]    messageId     The message id of the packet.
```

Comment:
What's a "message id"?  Is this the message counter?  If so, please call it that, both for the variable and in the text.

I'm aware that we have `PacketHeader::GetMessageId`.  We should fix that too, but that's out of scope for this PR.  Followup to do that?

Reply:
Renamed to messageCounter